src/Mimi.jl

```diff
@@ -571,6 +601,62 @@ function run_timestep(s::adder, t::Int)
     v.output[t] = p.input[t] + p.add[t]
 end
 
+#Begin Graph Functionality section
+
+function print_graph(m::Model)
+  for c in keys(m.nodes)
+    i_edges = get_connections(m,c,:INCOMING)
+    o_edges = get_connections(m,c,:OUTGOING)
+    println(c)
+    println("  incoming parameters:")
+    [println("    - ",e.target_parameter_name," from ",e.source_component_name) for e in i_edges]
+    println("  outgoing variables:")
+    [println("    - ",e.source_variable_name," from ",e.target_component_name) for e in o_edges]
+  end
+end
+
+function get_connections(m::Model, c::ComponentInstanceInfo, which::Symbol)
+  return get_connections(m, c.name, which)
+end
+
+function get_connections(m::Model, component_name::Symbol, which::Symbol)
+  #which = uppercase(which)
+  if which==:ALL
```

Comment:
Can you make these values lowercase, i.e. ``:all`` etc. For some reason that seems the julia convention. Btw., I think there might also be an enum type in julia that could be used here. I haven't really looked into that...